doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -243,6 +243,14 @@ To enable modem sleep and TAU pre-warning notifications, use the following optio
 
 For additional configurations related to these features, see the API documentation.
 
+Limitations
+***********
+
+* The LTE link control library and the :ref:`nrfxlib:nrf_modem_at` interface should be used at the same time with caution
+  because the library also uses the :ref:`nrfxlib:nrf_modem_at` interface.
```

Comment:
Can this sentence be rewritten in a different way? There is some repetition

Reply:
The sentence has been changed quite a bit.

Reply:
Done